docs/docs/guides/defining-config.md

```
@@ -37,17 +37,17 @@ Three pieces:
 class Advanced(Section):
     label = "Advanced Settings"
-    sort_order = 99  # shown last
+    sort_order = 99
 ```
 
 ## Field options
 
 | Parameter        | Type                     | Required | Description                                                         |
 | ---------------- | ------------------------ | -------- | ------------------------------------------------------------------- |
-| `frontend_model` | `type[BaseFrontendModel]`| Yes      | The field type class — determines Python type and UI widget. Full list [here](/reference/field-types/#overview)  |
+| `frontend_model` | `type[BaseFrontendModel]`| Yes      | The field type class — determines Python type and UI widget. Full list [here](/reference/field-types#overview)  |
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)